letsencrypt/constants.py

```diff
@@ -25,6 +26,18 @@
 """Defaults for CLI flags and `.IConfig` attributes."""
 
 
+RENEWER_DEFAULTS = configobj.ConfigObj(dict(
+    renewer_config_file="/etc/letsencrypt/renewer.conf",
+    renewal_configs_dir="/etc/letsencrypt/configs",
+    archive_dir="/etc/letsencrypt/archive",
+    live_dir="/etc/letsencrypt/live",
```

Comment:
The above directories should depend on `IConfig.config_dir` and `IConfig.work_dir` as supplied by the user. `letsencrypt --config-dir /tmp/le/config --work /tmp/le/work` will break the renewer. Hence the renewer script should symmetrically accept `letsencrypt-renewer --config-dir /tmp/le/config --work-dir /tmp/le/work`. Implementation is rather straightforward as it amounts to just a bit of copy&paste from `cli.py` and `configuration.Namespace`.